account_invoice_view_adj/views/account_invoice_view_adj.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <!-- Inherit the list view to add the 'Responsible' field -->
+        <record id="view_account_invoice_supplier_tree_inherit" model="ir.ui.view">
+            <field name="name">account.invoice.supplier.tree.inherit</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_supplier_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//tree/field[@name='state']" position="before">
+                    <!-- Adding the 'Responsible' field -->
```

Comment:
This comment is the same.

Reply:
Let it be known that adjustments are inherited from "account.invoice.supplier.tree", 
"Responsible"field name come from "account.invoice.supplier.form".
Refer to [135](https://www.quartile.co/forum/1/view-135)
I thought about it and decided comments wasn't necessary,
it's already written in the code.